src/command_modules/azure-cli-container/azure/cli/command_modules/container/_help.py

```diff
@@ -21,6 +21,8 @@
           text: az container create -g MyResourceGroup --name mywinapp --image winappimage:latest --os-type Windows --cpu 2 --memory 3.5
         - name: Create a container in a container group with public IP address and ports.
           text: az container create -g MyResourceGroup --name myalpine --image alpine:latest --ip-address public --ports 80 443
+        - name: Create a container in a container group with public IP address, ports and DNS name label.
+          text: az container create -g MyResourceGroup --name myalpine --image alpine:latest --ip-address public --ports 80 443 --dns_name_label contoso
```

Comment:
Should be `--dns-name-label` not `--dns_name_label`

Reply:
@derekbekoe , good catch. Fixed now.